packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -5,7 +5,7 @@ import clsx from 'clsx';
 import { elementTypeAcceptingRef } from '@material-ui/utils';
 import ownerWindow from '../utils/ownerWindow';
 import withForwardedRef from '../utils/withForwardedRef';
-import { setRef } from '../utils/reactHelpers';
```

Comment:
Why was the separation of these coupled methods needed? `setRef` is just a (bad) compat helper that shouldn't actually be used. The long term goal is to only export `useForkRef`.

Reply:
I was looking into the bundle size of the Textarea component. I have realized that clsx was imported in reactHelpers. It's a prehistoric solution for bundle size issue. Now, I believe it's a first good step in the right direction. We can flatten the methods of *helpers.js* and *reactHelpers.js*, basically, remove these two files.

Reply:
What do you think? If you agree with the direction but are not confortable with batching the changes in this pull request, I can isolate the efforts.

Reply:
If the change were to happen, I would put it in a separate PR.

Reply:
OK, I'm reverting it.